invokeai/app/services/workflow_records/workflow_records_sqlite.py

```diff
@@ -24,76 +29,286 @@ def __init__(self, db: SqliteDatabase) -> None:
 
     def start(self, invoker: Invoker) -> None:
         self._invoker = invoker
+        self._sync_default_workflows()
```

Comment:
What problem is this solving?

Reply:
Keeping the list of default (ie template) workflows in sync with whatever we want them to be

So invoke can change these along with the app and they are always up to date

I tried more efficient logic to only update changes ones but it got a bit edge case y. So just overwriting them fully on startup.

Note that this is only occurring in the SQLite implementation